Makefile

```diff
@@ -232,12 +232,15 @@ proto-image-push:
 	docker push $(protoImageName)
 
 ###############################################################################
-###                                Querygen                                 ###
+###                          Querygen and Modulegen                         ###
```

Comment:
What do you guys think about separating this? Or just calling this something like `Autogen`? Because if we add something new related to auto-generation, we will need to add it here as well